### DISCUSSION TITLE
機器人大戰

### DISCUSSION THREAD

**Discussion by 710388** [1101-GS4719-C/Python-Community-GS4719-C#2014] [優秀專案]
https://www.pollen-robotics.com/reachy/?gad_source=5&gclid=EAIaIQobChMIktfBzfP7hQMVFNMWBR0jggNmEAAYASAAEgLbcvD_BwE
不知道大家對機器人有沒有興趣，最近想要做一個類似上面網站的機器人，大家都會期待自己的機器人能有什麼功能呢？不妨告訴我，也許我會實現

**Comment by 155Julia** [comment]
我希望他是可以幫我掃地、拖地，做所有想得到家事的機器人，除了這個簡單的功能外，如果可以更進階，我希望他可以記得我的衣服褲子有哪些幫助我搭配，這樣出門就不用想太久。

**Comment by yubo0oo** [comment]
可以自己放音樂

**Comment by JustAHedgehog** [comment]
機器人的部分還有想出實際應用，然而個人想要有一台Foldimate的摺衣機器，可以節省摺衣的時間，甚至可以的話想要延伸至用吊掛式的方式放入機器就可以幫忙摺完衣服的那種。
目前推出的版本如下，相關介紹影片也一併奉上
![image](https://github.com/NCU-GS4719-Python/Python-Community/assets/143854000/1369296e-caf4-4aee-bcca-5a840f7d429a)

[介紹影片](https://www.youtube.com/watch?v=C76osXtpLeM)
